spacepy/pycdf/__init__.py

```diff
@@ -5919,7 +5699,7 @@ def type(self, name, new_type=None):
         @return: CDF variable type, see `pycdf.const`
         @rtype: int
         @note: If changing types, old and new must be equivalent, see CDF
-               User's Guide section 2.5.5 pg. 57
+        User's Guide section 2.5.5 pg. 57
```

Comment:
This should probably be either de-epydoced or just left alone.

Reply:
I reverted this, I will get rid of the epydocs in a later PR, I don't want to lost focus of what this PR is actually for.

Reply:
Ended up un-epydoccing this, it is resolved.